cmds/truncate/truncate.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Truncate - shrink or extend the size of a file to the specified size
+//
+// Synopsis:
+//     truncate [OPTIONS] [FILE]...
+//
+// Options:
+//     -s: size in bytes
+//     -c: do not create any files
+//
+// Author:
+//     Roland Kammerer <dev.rck@gmail.com>
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+var (
+	flags struct {
+		s int64
```

Comment:
Since you do not have duplicate flags here, you could just do

```go
var (
  create = flag.Bool("c", false, "Do not create files.")
  size = flag.Int64("s", -1, "Size in bytes")
)
```

Go usually prefers one-letter variable names, but only if it is *really* clear what it's referring to (e.g. `c := exec.Command(...)`). It's preferable to spell it out when that isn't the case, like here, esp if it's short anyway.